Report.md

```diff
@@ -129,29 +120,32 @@ Sprint Backlog | A subset of the Product Backlog for development in a sprint (ma
 Product Increment | A potentially shippable version of the product that builds on the previous increment/version
 Sprint Burndown Chart | A graphical representation of the development team's progress over time (backlog items completed)
 
-
 Workproducts and Models
 =======================
 6. Document your proposed workproducts and models *and descriptions*.
 
 ## **Use Case Diagrams**
+These diagrams help to visualise scenarios where there is interaction between users, the system, and other actors (namely Relationship Managers), describing what the system will do but not how it will be done. The diagrams below show this in regards to outbound and inbound calls respectively:
```

Comment:
Maybe instead of starting with these, we can start with the diagram itself (might be repetitive after the header but a bit more formal) So "Use case diagrams help to visualise.....". Also, can we add a part so it specifies that use case diagrams represent the expected behaviour of the interactions. So "Use case diagrams help to visualise the expected behaviour of the interacts between the users (customers), the system and other actions (namely Relationship Managers) and describes system behaviour but not how it will be done. Thus use case diagrams assists us in designing a system from the end user's perspective. The diagrams below show this in regards to outbound and inbound calls respectively: 

(I might have made this long - you can take stuff out)

Reply:
Totally happy to add that in - good point with the expected behaviour. Just in your example you still end with "The diagrams below" - But you reckon it's better at the bottom with like a ##Use case diagram description?
Only because it's the one description for 2 diagrams